### DISCUSSION TITLE
[日服][JP]問題点/改善点 Current Problems/Improvements[Ver5.5.0]

### DISCUSSION THREAD

**Discussion by wallsman** [MaaAssistantArknights/MaaAssistantArknights#2149] [General]
仕様（(technical) specification）
現段階ではほぼ目立つ仕様はなし
ほぼ現行の機能を使用可能。

Almost no noticeable specifications at this stage.
Almost all current features are available in bits and pieces.

現状の問題点（Current Issues）
【ツールボックス】
・OCRに難あり
ノイルホーンなど、異格と通常が区別できない。
使用FONTがおかしい？
![スクリーンショット 2024-08-09 17 37 51](https://github.com/user-attachments/assets/aeeebded-7365-4dcd-824f-b0eba28275ad)

【クラッシュ時のアークナイツ再起動ミス】
自環境のみ？
エミュレーター環境など併せて報告あれば助かります。

TODOでは対応予定記載有
https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/9853

他になにかあれば追加などお願いします。
If you have anything else to add, please do so.

**Comment by MistEO** [comment]
We plan to support Copilot (auto combat) function for all clients, and be compatible with existing CN json files!
